rosidl_generator_c/test/test_interfaces.c

```diff
@@ -896,3 +941,325 @@ int test_submessages(void)
 
   return 0;
```

Comment:
I don't think you need to change anything here.  The EXPECT_EQ macro already does a "return 1" if it fails; thus, we just need to return 0 at the end here in case everything passed.